UQ-eLIPSE/crucible-components
# Story/vethub 33

Added input to select the amount of questions a user wants.

To test:
- Go to https://localhost:5173/ and you should now see an input.
- Add a number to the input. You should notice that if you add something like 10, it will default to max
- Clicking the start button should show you the quiz with the amount of questions selected.

To run the component tests, run yarn test.
